hammer_cli_katello.gemspec

```diff
@@ -47,8 +47,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
   gem.version = HammerCLIKatello.version
 
-  gem.add_dependency 'hammer_cli', '~> 0.11.0'
-  gem.add_dependency 'hammer_cli_foreman', '~> 0.11.0'
+  gem.add_dependency 'hammer_cli', '~> 0.11'
+  gem.add_dependency 'hammer_cli_foreman', '~> 0.11'
```

Comment:
so apprentely I was wrong and hammer_cli_katello 0.11 is not compatible with hammer_cli 0.12, the gemspec should not have been updated, sorry (just confirmed with @mbacovsky)